### PR TITLE
Use CFPB brand colors in cf-expandables component

### PR DESCRIPTION
**Pull request by cfarm**
Adds CFPB brand colors as the default variable values for colors in `cf-expandables`. 

Depends on #561 - this includes changes from that branch in the diff.

## Testing

In this repo:
- check out this branch
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- Compare colors on local site: 
   - http://localhost:3000/components/cf-expandables/ 
   - to the live cf documentation to make sure they match: https://cfpb.github.io/capital-framework/components/cf-expandables/

### before (WITHOUT `cf-theme-overrides.less` colors)

screen shot 2017-08-18 at 2 24 29 pm
screen shot 2017-08-18 at 2 24 36 pm

### after (WITHOUT overrides stylesheet + with brand colors) 

note that fonts aren't loading locally so it looks slightly different from the docs:
screen shot 2017-08-18 at 2 36 17 pm
screen shot 2017-08-18 at 2 36 25 pm

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by contolini** [comment]
For some reason, when using Node 8, gulp doesn't compile cf-expandables' Less file. When using Node 6, running `npm run build` creates `/tmp/cf-expandables/cf-expandables.css`. With Node 8 that file is not created. I can't for the life of me figure out why. 😩  It results in the QUnit tests failing.

**Comment by anselmbradford** [comment]
Try rebasing with `canary` and running `npm run build && npm test` and see if there are any failures, then force push the branch back up here.

**Comment by cfarm** [comment]
Test still fails. I tried the following:

- merged canary
- deleted node_modules and package-lock.json
- `npm install`
- `npm run build`
- `npm test`

**Comment by anselmbradford** [comment]
> wonder if #578 will fix this

This is what I've done in my modals branch:
`git fetch && git checkout -b brand-colors-cf-expandables_webpack && rebase webpack_update`

**Comment by cfarm** [comment]
yup just about to try the same, will report back!

**Comment by cfarm** [comment]
still fails!! try it out on this branch: brand-colors-cf-expandables_webpack

Travis job: https://travis-ci.org/cfpb/capital-framework/builds/270935556?utm_source=github_status&utm_medium=notification

File: src/cf-expandables/src/cf-expandables.less [hunk 1]
```diff
@@ -7,34 +7,31 @@
...
+@expandable-group_header-text:  @gray;
+@expandable-group_header-bg:    @gray-10;
+@expandable-group-bg:           @white;
+@expandable-group-divider:      @gray-80;
 
 // Sizing variables
 
 // .o-expandable_link
-@expandable_link-font-size:     14px;
+@expandable_link-font-size:     @btn-font-size;
```
Comment (anselmbradford) [inline src/cf-expandables/src/cf-expandables.less]: The test failure problem is this: `@btn-font-size` is `undefined`.

**Comment by cfarm** [comment]
finally passing 🥇 🤣

**Pull request merged**